openedx/features/journals/templates/journals/journal_card.html

```diff
@@ -0,0 +1,45 @@
+<%def name="online_help_token()"><% return "course" %></%def>
```

Comment:
Just for my own learning, what is this line?  Should this be "journal"?  I haven't seen this pattern before

Reply:
removed, not sure why it was there